src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanelUtils.kt

```diff
@@ -17,34 +18,35 @@ fun CharSequence.isSimilarTo(sequence: CharSequence): Boolean {
     return levenshtein.apply(this, sequence).toDouble() / length < MINIMUM_EXAMPLES_SIMILARITY
 }
 
+fun getSmallInfoPaneContent(it: Any): String {
```

Comment:
It is better to move all of those ``getters'' to `object` or make all of them extensions functions. It is not idiomatic in Kotlin to create top-level functions that are, in reality, get :)

Reply:
Done